lib/util/token-store.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @fileoverview In-Memory Token Store
+ * @module box-node-sdk/token-store
+ */
+
+'use strict';
+
+var store = new Map();
+
+/**
+ * Basic in-memory Token Store, not suitable for use in production!
+ * @param {string} userID The ID of the user whose tokens will be stored
```

Comment:
Consider renaming; user ID is not necessary, any identifier will do.  For example, some applications use an internal session ID instead!